index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>theyalow</title>
+    <link rel="stylesheet" href="./style/style.css">
+
+</head>
+
+<body>
+    <div class="wrapper">
+        <main>
```

Comment:
a bit wrong using semantic tags, in the basic shell of the website sort of such structure should be:
1) header
2) main
3) footer
So "main" element shouldn't contain "header" or "footer" as a child. As for "nav" element, you can put it inside "main" or "header", it's logically, because application may contain a few "nav" panels